data-upgrade-news/src/main/java/org/exoplatform/news/upgrade/targets/PublishedNewsDisplayedPropUpgrade.java

```diff
@@ -53,8 +57,8 @@ public class PublishedNewsDisplayedPropUpgrade extends UpgradeProductPlugin {
 
   private PortalContainer          container;
 
-  private int                      publishedNewsDisplayedPropCount = 0;
-
+  private int                      migratedPublishedNewsCount = 0;
```

Comment:
no need to add a class attribute for this, in fact, the migrated news will simply be (BTW, a new log entry is proposed as well)  :
```java
    int migratedPublishedNewsCount = 0;
    int processedPublishedNewsCount = 0;
    int totalPublishedNewsCount = newsTargetsMetadataItems.size();
    for (List<MetadataItemEntity> newsTargetsMetadataItemsChunk : ListUtils.partition(newsTargetsMetadataItems, 10)) {
        int notMigratedPublishedNewsCountByTransaction = manageNewsTargetsMetadataItemsProps(newsTargetsMetadataItemsChunk);
        int processedPublishedNewsCountByTransaction = newsTargetsMetadataItemsChunk.size();

        processedPublishedNewsCount += processedPublishedNewsCountByTransaction;
        migratedPublishedNewsCount += processedPublishedNewsCountByTransaction - notMigratedPublishedNewsCountByTransaction;
        notMigratedPublishedNewsCount += notMigratedPublishedNewsCountByTransaction;
        LOG.info("News migration progress: processed={}/{} succeeded={} error={}", processedPublishedNewsCountByTransaction, migratedPublishedNewsCount, totalPublishedNewsCount, migratedPublishedNewsCount, notMigratedPublishedNewsCount);
    }
```